package.json

```diff
@@ -6,6 +6,10 @@
   "bin": {
     "alterschema": "bindings/node/cli.js"
   },
+  "scripts": {
+    "build": "webpack",
```

Comment:
We can already build by just doing`make www`. Let's remove this one and only keep `start`?